src/styles/card.scss

```diff
@@ -0,0 +1,84 @@
+$main-color: #00acdc;
+$white-color: #fff;
+$text-accent-color: #060b35;
+$text-secondary-color: #616070;
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
```

Comment:
It is a good practice to make all colors [variable](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)